.github/workflows/smoke-test.yml

```diff
@@ -124,7 +124,7 @@ jobs:
         run: |
           ./out/minikube start \
               --no-kubernetes \
-              --memory 2gb \
+              --memory 1200mb \
               --cpus 2 \
```

Comment:
In my tests on the intel runners, 1 cpu is most reliable configuration. But I did not test more than 1 cpu for long time, so maybe it works fine now.